static/app/components/searchQueryBuilder/index.tsx

```diff
@@ -40,6 +40,7 @@ export interface SearchQueryBuilderProps {
    * Indicates the usage of the search bar for analytics
    */
   searchSource: string;
+  autofocus?: boolean;
```

Comment:
Can we make this `autoFocus` instead? That's more consistent with what we have elsewhere